node/service/src/chain_spec.rs

```diff
@@ -157,7 +157,24 @@ pub fn rococo_config() -> Result<RococoChainSpec, String> {
 }
 
 pub fn versi_config() -> Result<VersiChainSpec, String> {
-	VersiChainSpec::from_json_bytes(&include_bytes!("../res/versi.json")[..])
+	let wasm_binary = rococo::WASM_BINARY.ok_or("Versi development wasm not available")?;
+
+	Ok(RococoChainSpec::from_genesis(
+		"Versi",
+		"versi_v1_3",
+		ChainType::Live,
+		move || RococoGenesisExt {
+			runtime_genesis_config: rococo_development_config_genesis(wasm_binary),
+			// Use 1 minute session length.
```

Comment:
Really one minute?

Reply:
Ah, damn I need to update the comment. 100 blocks is 10 minutes